### PR TITLE
Add indentation validation

### PR DESCRIPTION
**Pull request by promag**
This PR adds indentation validation and it correctly supports ES7.

Note that comments are not taken into account: https://github.com/eslint/eslint/issues/3845.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
-    indent: 'off',
+    indent: ['error', 2],
```
Comment (ruimarinho) [inline src/index.js]: Does this need VariableDeclarator or is the default 2 spaces?

Comment (ruiquelhas) [inline src/index.js]: Default is 4 spaces.

Comment (ruimarinho) [inline src/index.js]: ping @promag

Comment (promag) [inline src/index.js]: I don't have VariableDeclarator and it seems to correctly lint the code.

Comment (ruimarinho) [inline src/index.js]: @ruiquelhas can you confirm this and merge?

Comment (ruiquelhas) [inline src/index.js]: Aren't we using 1 space for `case` statements?

**Pull request merged**